spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/RootController.java

```diff
@@ -81,6 +81,7 @@ public RootController(EntityLinks entityLinks) {
 	@RequestMapping("/")
 	public ResourceSupport info() {
 		ResourceSupport resourceSupport = new ResourceSupport();
+		resourceSupport.add(new Link("/dashboard", "dashboard"));
```

Comment:
I know this is not strictly part of this PR, but we should try to limit the occurence of the magic string "/dashboard".
So maybe introduce a CONSTANT for it in `UiController` and reference that here, and also in `OAuthSecurityConfiguration` and `BasicAuthSecurityConfiguration`